examples/terraform/aws-dualstack/output.tf

```diff
@@ -0,0 +1,262 @@
+/*
+Copyright 2019 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+output "kubeone_api" {
+  description = "kube-apiserver LB endpoint"
+
+  value = {
+    endpoint                    = local.kubeapi_endpoint
+    apiserver_alternative_names = var.apiserver_alternative_names
+  }
+}
+
+output "ssh_commands" {
+  value = formatlist("ssh -J ${local.bastion_user}@${aws_instance.bastion.public_ip} ${local.ssh_username}@%s", aws_instance.control_plane.*.private_ip)
+}
+
+output "kubeone_hosts" {
+  description = "Control plane endpoints to SSH to"
+
+  value = {
+    control_plane = {
+      cluster_name         = var.cluster_name
+      cloud_provider       = "none" // intentionally set to none for kubelet --node-ip flag
```

Comment:
Since kubeone already supports external CCM for AWS, I think we should not be using `none` in this example. FYI, in KKP this is in progress: https://github.com/kubermatic/kubermatic/issues/11171

Reply:
Is using external CCM required for Dualstack on AWS?

Reply:
It is not required, but but it works better with the external CCM (with ext. CCM node IPs are properly populated in node status, and IPv6 load-balancer services work a bit better).

Reply:
@PratikDeoghare This must be `aws`.

Reply:
Yes. Yes. Testing that now.